week1_session4_intro_to_ml_and_scikit_learn.ipynb

```diff
@@ -0,0 +1,1547 @@
+{
```

Comment:
<p>I imagine that the work load here is quite high for one lesson. I would sort the exercises from "mandatory" to "optional", so that at the end of the lesson you can wrap up the core (and leave the rest for them to explore if interested).</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/volkamerlab/ai_in_medicine/pull/10/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='51'/>

Reply:
<p>I could leave everything after accuracy as optional, but I dont think its too bad. What do you think?</p>

Reply:
That sounds good! I also would not remove exercises, since some might be quick and interested.